### ISSUE TITLE
wo questions about this competition:

### ISSUE THREAD

**Issue by clearlove7-s11** [VIPriors/vipriors-challenges-toolkit#39]
Dear organizers, 
Hello, I finally need to consult you with two questions about this competition: 
1. 
When our team saw that the results could be hidden from the list, all the results of our team were hidden from the list, and now the final ranking does not have our results, we want to know what the final ranking will prevail? 
two。. 
Now the top 76 points on the list, we have good reason to suspect that they have used pre-training weights or other data sets. After all, the most advanced method scores only 84.5 on the complete K400 data set. How can anyone score 76 on the processed small sample K400? 
We are anxious to get your reply.
Yours
Marco

**Comment by clearlove7-s11** [comment]
Dear organizers,
Hello, I finally need to consult you with two questions about this competition:
1.When our team saw that the results could be hidden from the list, all the results of our team were hidden from the list, and now the final ranking does not have our results, we want to know what the final ranking will prevail?
2.We saw on the seminar's official website that the final deadline for all challenges was 23:59 GMT on September 24th, but when we tried to get the results back on the list within the specified time, we were told that the game was over. why did the server shut down early?
3.Now the top 76 points on the list, we have good reason to suspect that they have used pre-training weights or other data sets. After all, the most advanced method scores only 84.5 on the complete K400 data set. How can anyone score 76 on the processed small sample K400?
We are anxious to get your reply.
Yours
Marco
.




![IMG_8252](https://user-images.githubusercontent.com/58651191/134751615-49635d9a-13e0-45df-a43b-363e0ad09618.PNG)
![IMG_8255](https://user-images.githubusercontent.com/58651191/134751616-deafeb3b-f989-4631-b809-a88ffd307ea5.PNG)
<img width="476" alt="571768bc9e23273a90bb0eded9761b9" src="https://user-images.githubusercontent.com/58651191/134751619-8220810f-2658-42fc-b9df-a85417e8a793.png">
